src/main/java/com/example/flight/domains/Data.java

```diff
@@ -0,0 +1,15 @@
+package com.example.flight.domains;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.NoArgsConstructor;
+
+@lombok.Data
+@NoArgsConstructor
+@ApiModel(description = "Domains to model the data that is received form the FlightAPI")
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Data {
```

Comment:
`Data`? Data of what?

Reply:
This class is just to model the object from the FlightAPI and that's the name that they give to the response so the class should have the same name. A solution to the confusion of the name could be to add an explanation to the name in the documentation?